lib/ansible/modules/cloud/openstack/os_keystone_endpoint.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/python
+# Copyright (c) 2017 VEXXHOST, Inc.
```

Comment:
The [dev guide](http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#copyright) says otherwise

```
#!/usr/bin/python
# Copyright (c) 2017 Ansible Project
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Reply:
Yes, the dev guide did something different than a major rewrite did. I'm trying to get clarification on this.

Reply:
Ok, so we updated the dev guide to reflect the rewrite.
Now, this is not a deal-breaker, if you don't like or want it, I don't care :-)

Reply:
fixed and kudos for fixing the doc.